mlir/test/Dialect/LLVMIR/invalid.mlir

```diff
@@ -9,29 +9,38 @@ llvm.func @ctor() {
   llvm.return
 }
 
-// expected-error@+1{{mismatch between the number of ctors and the number of priorities}}
-llvm.mlir.global_ctors {ctors = [@ctor], priorities = []}
+// expected-error@+1{{ctors, priorities and data must have the same number of elements}}
```

Comment:
```suggestion
// expected-error@+1{{ctors, priorities, and data must have the same number of elements}}
```
ultra nit: sorry for the comma nit picking but I couldn't resist.

Johannes recently added this detailed explanation on an internal PR :)
![0_PrPsZb27MNQ8JzYx](https://github.com/user-attachments/assets/b2da4bc4-5099-474e-a001-f48baee850d0)

Reply:
hahah, epic explanation!